### ISSUE TITLE
NVDA not reporting expanded when collapsed link on website is pressed and changes state

### ISSUE THREAD

**Issue by bhavyashah** [nvaccess/nvda#12073]
You have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.
Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:
- [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
- [Feature request](https://github.com/nvaccess/nvda/issues/new?template=feature_request.md)

### Steps to reproduce:
* Go to http://catalog.ulm.edu/content.php?catoid=31&navoid=3902.
* Navigate to the ACCT 1010 - Accounting Survey collapsed link.
* Press Enter.

### Actual behavior:
Firefox: NVDA stays quiet and the section gets expanded revealing new content.
Edge: NVDA repeats the title of the link (ACCT 1010 - Accounting Survey in this example) and the section gets expanded revealing new content.

### Expected behavior:
NVDA should report "expanded" while the browser does its thing of expanding the section and revealing additional content. This behaviour should be consistent across different web browsers.

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.4 RC 1
#### Windows version:
10
#### Name and version of other software in use when reproducing the issue:
Firefox version 85.0.2
Microsoft Edge version 88.0.705.68
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Not tested, but presumably yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Did not try

**Comment by lukaszgo1** [comment]
ccc @jcsteh for the Firefox part.

**Comment by jcsteh** [comment]
This is a website authoring error. Rather than changing the expanded state on the link, the page removes the link from the document and replaces it with a new one. As far as screen readers are concerned, it is an entirely different link.

I'm not sure why the name gets read in Chromium.

Aside from this being fixed by the author, the only way to work around this in NVDA would be #8869, which got reverted some time ago. Even then, it would report the whole link, not just "expanded".

**Comment by Adriani90** [comment]
This is fixed in Firefox and works as expected in NVDA 2023.1 RC1 and Firefox 111. Closing as works for me.
